docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,498 @@
+| lcSuspend | suspend |
+| lcQuit | quit |
+
+For more information, see [basic events in addons](../../guides/applications/addon.md#event).
+
+In this file, each callback has only `console.log()` as implementation.
+
+To show a pop-up with a message, use `lcResume`. Remove all the lines related to other events except `lcResume`.
+
+![JavaScript file for resume](media/js_for_resume.png)
+
+You need to use `addonapi`, which includes APIs for addons. To show the pop-up, you can use `addonapi.window.showMessageBox()`. For example:
+
+```
+const addonAPI = require('addonapi');
+
+resume(winId) {
+    const options = {
+        type: 'info',
+        buttons: ['OK'],
+        title: 'Resume',
+        message: 'Hello. Resume occurred!!'
+    };
+    addonAPI.window.showMessageBox(winId, options);
+}
+```
+
+
+To verify whether the modified file works as expected, follow these steps:
+
+1. Using the changed file, build a signed package again.
+
+2. Before installing the newly built package, remove the installed addon.
+
+3. In Addon Settings App, to remove the installed addon, click **X** button (![addon settings x button](media/addon_settings_x_btn.png)).
+
+4. To install and run the newly built addon package, follow the [steps to run addon](#run-addon) again. 
+
+    The following pop-up apprears:
+
+    **Figure: Addon Pop-up on Resume Event in Basic UI**
+
+    ![Showing pop-up in Basic UI](media/hello_resume.png)
+
+
+<a name="load-html-from-addon"></a>
+### Load HTML file from addon
+
+In an addon, you can load your own HTML file. In this section, an example of an HTML file that shows `Hello, I'm an Addon` is explained.
+
+Ensure that all the files related to an addon are located under `shared\res` of your addon project. Therefore, the added HTML file must also be located in the same directory.
+
+To add a new HTML file for addon, unfold the BasicAddon in Project Explorer. Then, right-click on the shared/res > New > Files and enter the file name as `hello.html`.
+
+![Add a new file](media/new_file.png)
+
+![Generate the HTML file](media/generate_hello_html.png)
+
+Following is an example of `hello.html` HTML file:
+
+```
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, width=device-width">
+</head>
+<body>
+    <h1>Hello, I'm an Addon</h1>
+</body>
+</html>
+```
+
+The HTML file must be loaded in the addon. To do so, you can use `addonapi.window.loadURL()` when the resume event occurs. Following is the code for the `rusume()` event:
+
+```
+resume(winId) {
+    console.log('TmplDefault.resume');
+    addonAPI.window.loadURL(winId, 'file://' + __dirname + '/hello.html');
+    const options = {
+        type: 'info',
+        buttons: ['OK'],
+        title: 'TcResume',
+        message: 'Hello. Resume occurred!!'
+    };
+    addonAPI.window.showMessageBox(winId, options);
+}
+```
+
+Including the changes above, build a new package and install it.
```

Comment:
Yes. We need to remove the installed addon. I skipped to mention that because the readers had to experience it already in adding pop-up paragraph if they have followed the page from the start. 
My intention with the paragraph is, that readers should :
 1. build a new addon package including HTML file and codes to load the file
 2. remove installed addon
 3. install the newly built addon in step 1.

Reply:
@hyundukkim,

After the HTML file is loaded in the addon, follow these steps:

1. Build a new addon package including HTML file and codes to load the file.
2. Remove the installed addon.
3. Install the newly built addon.

The following screen appears:

Reply:
@surbhig12 ,

I want to change your suggestion as below:


To confirm whethere the HTML file is loaded in the addon, follow these steps:

1. Build a new addon package including HTML file and codes to load the file.
2. Remove the installed addon.
3. Install the newly built addon and run the Basic UI Web application.
    The following screen appears:

Please verify it.

Reply:
@hyundukkim,
Suggest changing to:

To verify whether the HTML file is loaded in the addon, follow these steps:

1. Build a new addon package including HTML file and codes to load the file.
2. Remove the installed addon.
3. Install the newly built addon and run the Basic UI Web application.
    The following screen appears:

Reply:
done